source/_posts/2024-12-19-voice-preview-edition-the-era-of-open-voice.markdown

```diff
@@ -0,0 +1,125 @@
+
+For some, our voice assistant is all they need; they just want to say a couple of commands, set timers, manage their shopping list, and control their most used devices. For others, we understand they want to ask their voice assistant to make whale sounds or to tell them how tall Taylor Swift is - this voice assistant doesn't entirely do those things ([yet](/voice_control/assist_create_open_ai_personality/)). We think there is still more we can do before this is ready for every home, and until then, we'll be selling this *Preview* of the future of voice assistants. We've built the best hardware on the market, and set a new standard for the coming years, allowing us to focus our development as we prepare our voice assistant for every home. Taking back our privacy isn't for everyone - it's a journey - and we want as many people as possible to join us early and make it better.
+
+### Built for Home Assistant
+
+Many other voice assistants work with Home Assistant, but this one was *built* for Home Assistant. Unlike other voice hardware that can work with Assist, this doesn't require flashing firmware or any assembly. You plug it into power, and it is seamlessly discovered by Home Assistant. A wizard instantly starts helping you set up your voice assistant, but critically, if you haven't used voice before, it will quickly guide you through what you need to get the best experience.
+
+<p class='img'><img src='/images/blog/2024-12-vpe/wizard.webp' alt="Video of Assist wizard">Get up and running with Voice Preview Edition in minutes with our new wizard</a>
+
+This is not a DIY product. We've worked to make the experience as smooth as possible, with easy and fast updates and settings you can manage from the Home Assistant UI.
+
+### Advanced audio processing
+
+If you have been following our work on voice, you know we've tried a lot of different voice assistant hardware. Most available Assist-capable hardware is bad at its most important job - hearing your voice and then providing audiovisual feedback. That was really what drove us to build Voice Preview Edition.
+
+<p class='img'><lite-youtube videoid="DS_8cDZKBPc" videotitle="Music Assistant - VPE"></lite-youtube>Voice Preview Editions mics and audio processors effortlessly hear commands through loud music it is playing</p>
+
+Our Assist software could only do so much with substandard audio, and its functionality is massively improved with clear audio. The dual microphones combined with the XMOS audio processing chip are what makes it so capable. Together, they allow Voice Preview Edition to have echo cancellation, stationary noise removal, and auto gain control, which all adds up to clearer audio. This combined with an ESP32-S3 with 8 MB octal PSRAM - one of the fastest ESP and RAM combinations available - makes for an incredibly responsive device. This is the best Assist hardware you can buy today, and it will continue to give a great experience as Assist's feature set expands in the years to come.
```

Comment:
```suggestion
Our Assist software could only do so much with substandard audio, and its functionality is massively improved with clear audio. The dual microphones combined with the XMOS audio processing chip are what makes it so capable. Together, they allow Voice Preview Edition to have echo cancellation, stationary noise removal, and auto gain control, which all adds up to clearer audio. This combined with an ESP32-S3 with 8 MB of octal PSRAM - one of the fastest ESP and RAM combinations available - makes for an incredibly responsive device. This is the best Assist hardware you can buy today, and it will continue to give a great experience as Assist's feature set expands in the years to come.
```